CustomScript/customscript.py

```diff
@@ -594,5 +592,20 @@ def get_formatted_log(summary, stdout, stderr):
     return msg_format.format(summary, stdout, stderr)
 
 
+def get_command_to_execute(hutil):
+    public_settings = hutil.get_public_settings()
+    protected_settings = hutil.get_protected_settings()
+    cmd_public = public_settings.get('commandToExecute')
+    cmd_protected = protected_settings.get('commandToExecute')
```

Comment:
The design is:
1. The protected settings field will have the same commandToExecute naming
2. If customer provide both then we will fail and not process the config and properly message the failure in status channel